include/vulkan/vk_dispatch_table_helper.h

```diff
@@ -1,2354 +0,0 @@
-// *** THIS FILE IS GENERATED - DO NOT EDIT ***
-// See dispatch_table_helper_generator.py for modifications
-/***************************************************************************
-*
-* Copyright (c) 2015-2023 The Khronos Group Inc.
-* Copyright (c) 2015-2023 Valve Corporation
-* Copyright (c) 2015-2023 LunarG, Inc.
-* Copyright (c) 2015-2023 Google Inc.
-*
-* Licensed under the Apache License, Version 2.0 (the "License");
-* you may not use this file except in compliance with the License.
-* You may obtain a copy of the License at
-*
-*     http://www.apache.org/licenses/LICENSE-2.0
-*
-* Unless required by applicable law or agreed to in writing, software
-* distributed under the License is distributed on an "AS IS" BASIS,
-* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-* See the License for the specific language governing permissions and
-* limitations under the License.
-****************************************************************************/
-// NOLINTBEGIN
-#pragma once
-
-#include <vulkan/vulkan.h>
-#include <vulkan/vk_layer.h>
-#include <cstring>
-#include <string>
-#include "vk_layer_dispatch_table.h"
-#include "vk_extension_helper.h"
```

Comment:
vk_extension_helper.h refers to a file that only exists in vvl which in turn refers to even more custom containers